packages/swirl-components/src/components/swirl-shell-layout/swirl-shell-layout.css

```diff
@@ -134,22 +135,19 @@
   display: flex;
   overflow: hidden;
   align-items: center;
+  height: 4.5rem;
```

Comment:
**issue:** Logo and button aren't centered.

<img width="275" alt="Bildschirm­foto 2023-03-06 um 09 04 32" src="https://user-images.githubusercontent.com/1019414/223052401-0d4555b0-8d77-41fb-8ee8-84b0c0e7ad1c.png">